.github/test-infra/aws/rke2/terraform.tfvars

```diff
@@ -0,0 +1,8 @@
+default_user = "ec2-user"
+ssh_key_name = "packer-rke2-rhel-key"
+os_distro    = "rhel"
+# Need to allow in from internet for github runner to connect to node
+allowed_in_cidrs = ["0.0.0.0/0"]
+
+ami_id       = "ami-0699e51c9b070e3d5"
+rke2_version = "1.30"
```

Comment:
Food for thought - might be good to think through a way to grab the ami id with a lookup to auto-update to the latest AMI for a given RKE2 version. I think `ami-06510b3e0431a5ad3` is the newest one, published a few days ago [here](https://github.com/defenseunicorns/uds-rke2-image-builder/actions/runs/11119553607/job/30894933270#step:7:2957). This current one is from August if I'm reading the list right.

Reply:
Via the web console I filtered on AMI name containing `uds-rhel-rke2-v1.30` and then was able to sort by creation date. I think we could do similar with TF and use the rke2_version var here as part of the filter.

Reply:
Added in https://github.com/defenseunicorns/uds-core/pull/808/commits/062a88b70eb10087620d1ec181bf2e83974e1759